docs/_documentations/mdt-che-setupregistries.md

```diff
@@ -1,25 +1,75 @@
 ---
 layout: docs
-title: Installing Codewind on the Cloud
-description: Installing Codewind on the Cloud
-keywords: build, deploy, IBM Cloud Private, install, installing, installation, chart, Helm, develop, cloud, public cloud, services, command line, cli, command, start, stop, update, open, delete, options, operation, devops, OpenShift, OKD
+title: Adding registries in Che
+description: Adding registries in Che
+keywords: build, deploy, install, installing, installation, chart, Helm, develop, cloud, public cloud, services, command line, cli, command, start, stop, update, open, delete, options, operation, devops
 duration: 1 minute
 permalink: mdt-che-setupregistries
 type: document
-order: 20
-parent: root
 ---
 
-### Adding registries in Che
-After Che is started and running, add the container registry that will be used with Codewind.
-- On OpenShift or other Kube platforms, you can push your images to any Docker registry, such as Docker Hub, Quay.io, Google Cloud Registry (GCR), and more.
-- On IBM Cloud Private, you can push your image to any Docker registry *except* the internal Docker registry.
+# Adding registries in Che
+After Che is started and running, add the container registry to be used with Codewind.
+1. From the Codewind Project Explorer view, right-click **Projects** and select **Image Registry Manager**.
+2. From the **Image Registry Manager**, click **Add New**.
+3. On OpenShift and other Kubernetes platforms, push your images to a container registry, such as Docker Hub, Quay.io, Google Cloud Registry (GCR), and more. Enter the following information to add the registries:
```

Comment:
Change this to something like: `Enter the following information to add the registries`